server/app/controllers/application_controller.rb

```diff
@@ -2,5 +2,9 @@ class ApplicationController < ActionController::API
   #protect_from_forgery with: :exception
   include Response
   include ExceptionHandler
+
+  def set_default_response_format
+    request.format = :json
```

Comment:
it looks strange the method name refers to response while format for request is set - is this intended?